realm/realm-library/src/syncIntegrationTest/java/io/realm/PermissionManagerTests.java

```diff
@@ -159,6 +160,80 @@ public void onError(ObjectServerError error) {
 
     @Test
     @RunTestInLooperThread
+    public void getDefaultPermissions_returnLoadedResults() {
+        PermissionManager pm = user.getPermissionManager();
+        looperThread.closeAfterTest(pm);
+        pm.getPermissions(new PermissionManager.Callback<RealmResults<Permission>>() {
```

Comment:
I think if this test's name is correct, it should be `pm.getDefaultPermissions`. Otherwise, its test's name should be `getPermissions_returnLoadedResults`.